### ISSUE TITLE
Installation error 

### ISSUE THREAD

**Issue by rzvg** [abap2xlsx/abap2xlsx#1052]
Hi all,
I tried to update to the current version and got the following error:

Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_ALV failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_ALV_GRID failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_RESULT failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_RESULT_EX failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_RESULT_WD failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_CONVERTER_SALV_TABLE failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZCL_EXCEL_OLE failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZABAP2EXCEL_NOT_CLOUD failed 
Application component 'HLB0009009' is not allowed (see long text) 
Import of object ZEXCEL_TEMPLATE_GET_TYPES failed 

Does someone have any idea how to get teh missing calsses into the system?

Thank you very much!

Regards,
Veit

**Comment by sandraros** [comment]
I think it's a completely-different issue, not related to abap2xlsx (nor abapGit). The message `Application component HLB0009009 is not allowed` probably means that the corresponding abap2xlsx package is linked to the `BC` application component, but as far as I know, abap2xlsx linked its packages to no application component at all. Could you check in your system if you changed the application component of the abap2xlsx packages?

**Comment by rzvg** [comment]
I created a package ZABAP2EXCEL some years ago and now I want to update the ABAP2XLSX component.
The package has application component BC and software component HOME.
I downloaded teh ZIP from abap2xlsx and tried to import it into our SAP T01 system.
When I click on "Pull ZIP", all classes where updated , except the ones mentioned in the above error (e.g.  ZCL_EXCEL_CONVERTER is missing now.)
The long text of teh error is:
Application component 'HLB0009009' is not allowed (see long text)
Message no. PAK010

Diagnosis
Application component 'HLB0009009' is either not allowed for development packages because it cannot be used for assigning in CSS messages or it is not allowed for software component HOME.

So it seems, that some of the Z-classes can be created in my package.
